src/index.md

```diff
@@ -1,8 +1,10 @@
-# Welcome
+# Welcome to Deis Workflow
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon Kubernetes to provide a lightweight PaaS with a Heroku-inspired workflow.
+Deis (pronounced DAY-iss) Workflow is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis Workflow adds a devleoper-friendly Platform as a Service (PaaS) layer to any Kubernetes cluster. Deis Workflow provides a simple REST API, application log aggregation, release and rollback, edge routing, source to image, and authentication and authorization.
```

Comment:
Typo: "developer-friendly"

Also "source to image" didn't flow when I read this. Could we say "source code to Docker image"?